docs/source/conf.py

```diff
@@ -24,7 +28,6 @@
 PATH_ROOT = os.path.dirname(os.path.dirname(PATH_HERE))
 sys.path.insert(0, os.path.abspath(PATH_ROOT))
 
-import rising  # noqa: E402
```

Comment:
This has to stay!

Reply:
hmm, I did not move it, maybe some of the pre-commit hooks